### PR TITLE
136 Single Number

### PR DESCRIPTION
**Pull request by nayuta-ai**

### CODE REVIEW

File: 136_Single_Number.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# O(n)
+import collections
```
Comment (tkkawa) [inline 136_Single_Number.py:2]: from collections import Counter

File: 136_Single_Number.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+class Solution:
+    def singleNumber(self, nums: List[int]) -> int:
+        c = collections.Counter(nums)
```
Comment (tkkawa) [inline 136_Single_Number.py:5]: c = Counter(nums)

**Review by tkkawa — COMMENTED** [review]
write both time complexity and space complexity

**Review by tkkawa — CHANGES_REQUESTED** [review]
write both time complexity and space complexity

**Review by tkkawa — APPROVED** [review]
LGTM

File: 136_Single_Number.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# time O(n)
```
Comment (tkkawa) [inline 136_Single_Number.py:1]: clarify what n is

**Review by tkkawa — CHANGES_REQUESTED** [review]
clarify what n is

File: 136_Single_Number.py (hunk 3 above)
Comment (nayuta-ai) [reply 136_Single_Number.py]: describe what n is

**Review by tkkawa — APPROVED** [review]
LGTM

**Pull request merged**